src/SignalR/clients/ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts

```diff
@@ -27,7 +27,7 @@ export class MessagePackHubProtocol implements IHubProtocol {
 
     /** Creates an array of HubMessage objects from the specified serialized representation.
      *
-     * @param {ArrayBuffer | Buffer} input An ArrayBuffer containing the serialized representation.
+     * @param {ArrayBuffer | Buffer} input An ArrayBuffer or Buffer containing the serialized representation.
```

Comment:
@BrennanConroy Why didn't make input `string | ArrayBuffer | Buffer` and match IHubProtocol exactly? Why don't you get an error like `Type 'string | ArrayBuffer | Buffer' is not assignable to type 'ArrayBuffer | Buffer'`?